include/fast_float/ascii_number.h

```diff
@@ -282,38 +286,39 @@ report_parse_error(UC const *p, parse_error error) {
 template <typename UC>
 fastfloat_really_inline FASTFLOAT_CONSTEXPR20 parsed_number_string_t<UC>
 parse_number_string(UC const *p, UC const *pend,
-                    parse_options_t<UC> options) noexcept {
-  chars_format const fmt = detail::adjust_for_feature_macros(options.format);
-  UC const decimal_point = options.decimal_point;
+                    const parse_options_t<UC> options) noexcept {
 
   parsed_number_string_t<UC> answer;
   answer.valid = false;
   answer.too_many_digits = false;
-  // assume p < pend, so dereference without checks;
+  [[assume(p < pend)]]; // assume p < pend, so dereference without checks;
```

Comment:
Assume C++11 compatibility

Reply:
Hmm, maybe I don't understand correctly, but it's standard since C++23 https://en.cppreference.com/w/cpp/language/attributes/assume and older versions is produced a warning. I need to place this construction into macros, that depends on the platform?

Reply:
Yes, the assume attribute is C++23, but this library supports C++11, so that attribute cannot be used.